orbs/phraseapp/utils/autocreate-phraseapp-pr

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+# This script creates a pull request on GitHub with updated locales from PhraseApp,
+# bypassing the need for human intervention via PhraseApp "GitHub Sync" feature.
+
+git config --global user.email $PHRASEAPP_USER_EMAIL
+
+# Delete a phraseapp-locales branch if it exists
+echo "Deleting the phraseapp-locales branch if it exists..."
+git push origin --delete phraseapp-locales
+
+
+## Download the newest locale files from PhraseApp
+echo "Downloading the latest locale files from PhraseApp..."
+
+locales=($(yq e '.phrase.available_locales' .phraseapp.yml))
+pull_target_file_format=$(yq e '.phrase.pull.targets.0.params.file_format' .phraseapp.yml)
+pull_target_file_path=$(yq e '.phrase.pull.targets.0.file' .phraseapp.yml | sed s/"<locale_name>.*"//)
+
+for locale in "${locales[@]}"
+do
+  curl -H "Authorization: token $PHRASEAPP_ACCESS_TOKEN" \
+    "https://api.phrase.com/v2/projects/$PHRASEAPP_PROJECT_ID/locales/$locale/download?file_format=$pull_target_file_format" \
```

Comment:
Why don't you use the [phraseapp cli](https://phrase.com/cli/) which uses the default file [.phrase.yml](https://github.com/jobteaser/jobad/blob/master/.phrase.yml) ? Projects can configure it and the orb can only use: `phraseapp pull`.
I see 2 downsides of using the REST API:
- if we want to use a phraseapp feature,(e.g recently @BenjaminDigeon recently used the [tags feature](https://developers.phrase.com/api/#get-/projects/-project_id-/locales/-id-/download) in [jobad ](https://github.com/jobteaser/jobad/blob/master/.phrase.yml#L7).. With the current ORB implementation, we have to update the query params to add `tags[]` + the configuration file + documentation. Whereas the cli handles it automatically from the default `.phrase.yml`
- you have to specify all the locales. We have 13 locales (maybe more coming) in jobteaser and I don't see situations where we want to cherry pick them :thinking: Maybe you can use the [API](https://developers.phrase.com/api/#get-/projects/-project_id-/locales) to fetch all of them.

Reply:
@nicosvirjt, thanks for the feedback. 
I was not aware of this possibility. I just added the `phrase-cli `, and I think it's better this way. Thanks 🙂